cachi2/core/models/input.py

```diff
@@ -73,6 +73,7 @@ class BundlerPackageInput(_PackageInputBase):
     """Accepted input for a bundler package."""
 
     type: Literal["bundler"]
+    allow_binary: bool = False
```

Comment:
**Commit message:**

I don't think this is a "fix" anymore. By handling this with a CLI flag it is a genuine feature ;) and so the commit subject should be adjusted to actually match the vibe.